### PR TITLE
style, fix: Several redesigns and bug fixes.

### PR DESCRIPTION
**Pull request by thisisshadow**

### CODE REVIEW

**Comment by ilyassesalama** [comment]
Great improvements, I've tested your changes and everything seems good so far. However, more in depth review of your pull request is still needed since you have done a lot of changes here and there. Kindly, don't close this pull request until we review it.

File: app/src/main/java/a/a/a/kv.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+import com.besome.sketch.beans.ProjectLibraryBean;
+import com.besome.sketch.editor.manage.library.firebase.FirebaseActivity;
+import com.google.android.material.materialswitch.MaterialSwitch;
+import com.sketchware.remod.R;
+
+import mod.SketchwareUtil;
+import mod.hey.studios.util.Helper;
+
+public class kv extends LinearLayout implements nv {
```
Comment (PranavPurwar) [inline app/src/main/java/a/a/a/kv.java:17]: Hey, it seems that you have decompiled this class. It'd be better if you first commit the decompiled file (without any changes) and refactor/fix it in another separate commit. That makes it easier to debug issues that might have been caused by decompiling later on. 
Thanks

**Comment by devsuay** [comment]
check pull requests @thisisshadow

**Comment by hasrat-ali** [comment]
@coderabbitai review

**Comment by thisisshadow** [comment]
It has been waiting for approval for about 2 weeks 😑

**Comment by PranavPurwar** [comment]
The changes are massive (97 changed files) and the current modders don't have much free time rn, so that's probably why

**Comment by thisisshadow** [comment]
> The changes are massive (97 changed files) and the current modders don't have much free time rn, so that's probably why

Okay bro

**Comment by ilyassesalama** [comment]
I just got the time to review your pull request. Your changes are outstanding, thank you @thisisshadow for your contribution. We hope to see more contributions from you, the community will appreciate them.

**Comment by thisisshadow** [comment]
> I just got the time to review your pull request. Your changes are outstanding, thank you @thisisshadow for your contribution. We hope to see more contributions from you, the community will appreciate them.
> 
> I have one note about your commits, can you separate them next time instead of committing many files under one title even if the changes are not related to what you've named your commit. Separating them would help with reviewing your pull requests.

Thank you bro, next time I will do as you say.

**Pull request merged**